lib/mix/tasks/thesis.install.ex

```diff
@@ -49,6 +49,7 @@ defmodule Mix.Tasks.Thesis.Install do
 
     all_templates
     |> Stream.map(&render_eex/1)
+    |> Stream.map(&replace_yourapp(&1, to_string(Mix.Phoenix.otp_app))) # TODO: better way to infer app path?
```

Comment:
This is how we figure out where to put the `thesis_auth.ex` file in our `mix thesis.install` script.